### PR TITLE
Develop carey

### PR DESCRIPTION
**Pull request by DanCarey404**
Added TimeRelatedChanges_v11.md

### CODE REVIEW

File: migration/v11.0/TimeRelatedChanges_v11.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+The nature of these changes and the forms your data make take mean it is impossible to provide scripted code for data conversion.  Instead, we offer this guidance to assist you.
+
+Replacing gist:TimeInstant instances, gist:hasActual, gist:hasPlanned, gist:hasStart, gist:hasEnd
+-----
+
+To accommodate the deletion of the `TimeInstant` class and its associated object properties, we have added a number of new sub-properties to the `gist:atDateTime` data property.  The new datatype sub-properties serve two different functions.
+
+First, they allow you to explicitly call out the precision of the object data: date, year, minute, or microsecond.  The selection of precision is your choice.  But note that all the properties take the same `xsd:dateTime` range.  Any enforcement of your desired data precision will be up to you.
+
+Second, the new sub-properties give you the semantic distinctions between atart and end date-times, and planned vs actual.
```
Comment (uscholdm) [inline migration/v11.0/TimeRelatedChanges_v11.md:17]: Typo: 'atart' --> 'start'

Second is really second and third, they are quite different things.

**Review by uscholdm — CHANGES_REQUESTED** [review]
This is very good material. I did not have a chance to read it in detail.  I hope to get to that soon.
One initial thought is to simplify the presentation of the example. It's a bit confusing explaining all the decisions to be made. I suggest to start with one straightforward conversion. Then say, by the way, this assumes that you want X. But if you want Y, you would need to do it this other way.  Do that for each of the decisions. then summarize at the end.

**Review by rjyounes — APPROVED** [review]

**Comment by rjyounes** [comment]
I fixed the typo. The review came in too late to make substantive changes, as the PR had to be ready for merge by EOD yesterday.

**Comment by uscholdm** [comment]
That makes sense.

**Comment by rjyounes** [comment]
@uschold We can consider substantive changes in the next release. You could enter an issue if you want to pursue those.

**Pull request merged**